tests/imodels-clients-tests/src/integration/management/IModelOperations.test.ts

```diff
@@ -336,6 +336,72 @@ describe("[Management] IModelOperations", () => {
     });
   });
 
+  [
+    // {
+    //   label: "without changeset id/index specified",
+    //   getChangesetIdOrIndex: (): ChangesetIdOrIndex | undefined => undefined,
+    //   getExpectedChangesetCount: () => testIModelFileProvider.changesets.length
+    // },
+    // {
+    //   label: "with empty changeset id specified",
+    //   getChangesetIdOrIndex: (): ChangesetIdOrIndex | undefined => ({ changesetId: "" }),
+    //   getExpectedChangesetCount: () => 0
+    // },
```

Comment:
let's not keep commented out code. I think we can keep one with `changesetId` and `changesetIndex` (both uncommented), the rest should be deleted.

Reply:
removed it